internal/services/appservice/linux_web_app_slot_resource_test.go

```diff
@@ -2469,8 +2484,10 @@ provider "azurerm" {
 %s
 
 resource "azurerm_linux_web_app_slot" "test" {
-  name           = "acctestWAS-%d"
-  app_service_id = azurerm_linux_web_app.test.id
+  name                                           = "acctestWAS-%d"
+  app_service_id                                 = azurerm_linux_web_app.test.id
+  ftp_publish_basic_authentication_enabled       = false
+  webdeploy_publish_basic_authentication_enabled = false
```

Comment:
Why are these being set in the php template?

Reply:
I want to see how the tests come up. I think it was because those get set by default when you create it, and when I Was running the tests, it was failing because of these.

Reply:
The PHP tests are passing on our side without these properties, so these should also be removed 
```suggestion
```

Reply:
Thanks @stephybun missed that one. Should be done now :)